### PR TITLE
api delete post

### PR DESCRIPTION
**Pull request by ghost**
## Description
api delete post
## Reference Requirements
- Task ID: TI-1

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

### CODE REVIEW

**Comment by at-sonle** [comment]
LGTM

File: app/Http/Controllers/Api/PostController.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function destroy(Post $post)
+    {
+        if ($post->user_id == Auth::id()) {
+            try {
+                $post->delete();
+                return $this->responseDeleteSuccess(Response::HTTP_OK);
+            } catch (ModelNotFoundException $e) {
```
Comment (at-tuanphan) [inline app/Http/Controllers/Api/PostController.php]: vì sao phải phóng ra not found em?

**Review by ghost — COMMENTED** [review]
please check my comment

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**